cmake/ccache.cmake

```diff
@@ -20,6 +20,18 @@ if ($ENV{GRPC_BUILD_ENABLE_CCACHE})
     message(STATUS "Will use ccache as compiler launcher: ${gRPC_CCACHE_BINARY}")
     set(CMAKE_C_COMPILER_LAUNCHER   ${gRPC_CCACHE_BINARY})
     set(CMAKE_CXX_COMPILER_LAUNCHER ${gRPC_CCACHE_BINARY})
+
```

Comment:
Out of curiosity, how does this pdb race condition happen? PDB is usually created at the linking time and linking is done by a single process.

Reply:
AFAIK msvc by default has different behavior by default. Instead of keeping the debug info in .obj files, it creates a single PDB file that acts as a "database"  of all the debug symbols for a given compilation unit. 
That doesn't seem to play well with ccache.
Also see the documentation for sccache [here](https://github.com/mozilla/sccache#usage) that mentions this problem and the cmake workaround.